Check all errors

I'm a big fan of static analysis tools like [honnef.co/go/tools/cmd/staticcheck](https://github.com/dominikh/go-tools/tree/master/cmd/staticcheck)

Running this tool, I found 2 unchecked errors:
```
runner/solr-runner.go:47:11: this value of err is never used (SA4006)
solr_test.go:328:14: this value of err is never used (SA4006)
```

--- a/runner/solr-runner.go
+++ b/runner/solr-runner.go
@@ -3,11 +3,12 @@ package main
 import (
 	"crypto/rand"
 	"fmt"
-	. "github.com/sendgrid/go-solr"
 	"io"
 	"os"
 	"strconv"
 	"time"
+
+	. "github.com/sendgrid/go-solr"
 )
 
 var (
@@ -45,6 +46,9 @@ func main() {
 		limit, err = strconv.Atoi(os.Args[0])
 		if err != nil {
 			limit, err = strconv.Atoi(os.Args[1])
+			if err != nil {
+				panic(fmt.Sprintln("could not convert limit to int"))
+			}
 		}
 	}
 	numFound, err := run(limit, "/3")

--- a/solr_test.go
+++ b/solr_test.go
@@ -326,6 +326,7 @@ var _ = Describe("Solr Client", func() {
 			err = solrNoAuthClient.Listen()
 			Expect(err).To(BeNil())
 			replicas, err := locator.GetReplicaUris()
+			Expect(err).To(BeNil())
 			r, err := solrNoAuthHttp.Select(replicas, solr.FilterQuery("*:*"), solr.Rows(10))
 			Expect(err).To(Not(BeNil()))
 			Expect(strings.Contains(err.Error(), "401")).To(BeTrue())

